services/app-api/src/emailer/emailer.ts

```diff
@@ -124,27 +124,27 @@ type Emailer = {
         contract: ContractRevisionType,
         submitterEmails: string[],
         statePrograms: ProgramType[],
-        question: Question
+        question: ContractQuestionType
     ) => Promise<void | Error>
     sendQuestionsCMSEmail: (
         contract: ContractRevisionType,
         stateAnalystsEmails: StateAnalystsEmails,
         statePrograms: ProgramType[],
-        questions: Question[]
+        questions: ContractQuestionType[]
```

Comment:
FYI @pearl-truss  since you mentioned working on emailer - types changes incoming